clojure_liberator/src/clojure_liberator/data_access.clj

```diff
@@ -0,0 +1,22 @@
+(ns clojure-liberator.data-access)
+(use 'korma.db)
+(use 'korma.core)
+(require '[clojure.string :as str])
```

Comment:
Good idea, I should see if there is a good Haskell style guide out there somewhere.